build.js

```diff
@@ -242,6 +243,16 @@ function updateConfigPlist(app, config, plist) {
     });
 }
 
+function addExtrascript(app, path) {
```

Comment:
what is this for ?

Reply:
Considering a situation where we have to run extra script in the build phase.It would take the content of shell from manifest and put in the **extra** shell script.